libraries/aws_cloudformation_stacks.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsCloudformationStacks < AwsResourceBase
+  name 'aws_cloudformation_stacks'
+  desc 'Verifies settings for aws CloudFormation Stacks'
+
+  example "
+    describe aws_cloudformation_stacks do
+      it { should exist }
+    end
+  "
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:names, field: :name)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    cloudformation_stacks_rows = []
+    catch_aws_errors do
+      @api_response = @aws.cloudformation_client.describe_stacks
```

Comment:
This [request](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudFormation/Client.html#describe_stacks-instance_method) is a [pageable response](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/PageableResponse.html), you would have to paginate this request

Reply:
on it..